### PR TITLE
Remove unused @ts-expect-error directive

### PR DESCRIPTION
**Pull request by kamilmielnik**
image

### CODE REVIEW

File: e2e/test/scenarios/dashboard-filters/dashboard-filters-clear-and-restore.cy.spec.ts [hunk 1]
```diff
@@ -1,27 +1,20 @@
 import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
 import {
+  checkFilterListSourceHasValue,
+  createQuestionAndDashboard,
   editDashboard,
+  modal,
```
Comment (nemanjaglumac) [inline e2e/test/scenarios/dashboard-filters/dashboard-filters-clear-and-restore.cy.spec.ts:6]: WDYT about auto formatting in ascending order like we do for the regular js/ts imports?
Maybe a good project for devex?

**Review by nemanjaglumac — APPROVED** [review]

**Review by romeovs — APPROVED** [review]

File: e2e/test/scenarios/dashboard-filters/dashboard-filters-clear-and-restore.cy.spec.ts (hunk 1 above)
Comment (kamilmielnik) [reply e2e/test/scenarios/dashboard-filters/dashboard-filters-clear-and-restore.cy.spec.ts:6]: Yes, please :pray: cc: @uladzimirdev

**Comment by uladzimirdev** [comment]
@kamilmielnik why typescript didn't catch it earlier? in the code if you have unused ts-expect-error, it will throw TS error

File: e2e/test/scenarios/dashboard-filters/dashboard-filters-clear-and-restore.cy.spec.ts (hunk 1 above)
Comment (uladzimirdev) [reply e2e/test/scenarios/dashboard-filters/dashboard-filters-clear-and-restore.cy.spec.ts:6]: done https://github.com/metabase/metabase/pull/47009

**Pull request merged**